gravitee-apim-e2e/api-test/src/imports/api-import-with-metadata.spec.ts

```diff
@@ -105,7 +106,7 @@ describe('API - Imports with metadata', () => {
   });
 
   afterAll(async () => {
-    const deletedApis = [firstExpectedApiId, secondExpectedApiId, thirdExpectedApiId].map(
+    const deletedApis = [firstExpectedApiId, secondExpectedApiId, thirdExpectedApiId, fourthExpectedApiId].map(
```

Comment:
This kind of forgotten makes some test flaky (depending on the execution order) 😁